services/minespace-web/src/constants/permissions.js

```diff
@@ -0,0 +1,2 @@
+export const IN_DEVELOPMENT = "inDevelopment";
+export const IN_TESTING = "inTesting";
```

Comment:
These constants in this file and in Core's version don't seem to be used anywhere. Perhaps they're meant to be used in the `const authorizationGuard = ...` in AuthorizationGuard.js?

Reply:
Made it so they're used, modified Core to use them as well.